addons/feature-unshared/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Feature unshared projects",
+  "name": "Unshared featured projects",
```

Comment:
Verb is better IMO

Reply:
Yeah. Unless we go for something very explicit and long like `Show unshared projects when changing featured project`